addons/attach/CfgVehicles.hpp

```diff
@@ -86,6 +86,10 @@ class CfgVehicles {
             dayLight = 0;
             onlyInNvg = 1;
             useFlare = 0;
+            maxLifetime = 28800; // 8 hours
```

Comment:
```suggestion
            maxLifetime = 8 * 60 * 60;
```
?
Does the engine even respect high lifetimes on this simulation type?

Reply:
It should in the next update. https://feedback.bistudio.com/T158909

Reply:
Shouldn't this be quoted?